tests/filecheck/mlir-conversion/with-mlir/dialects/pdl_interp/pdl_interp_ops.mlir

```diff
@@ -107,7 +109,9 @@ module @rewriters {
 // CHECK-NEXT:     ^15:
 // CHECK-NEXT:       pdl_interp.record_match @rewriters::@pdl_generated_rewriter(%5, %3, %7, %4, [[arg0]] : !pdl.value, !pdl.value, !pdl.type, !pdl.value, !pdl.operation) : benefit(1), generatedOps(["arith.subi", "arith.addi"]), loc([%2, [[arg0]]]), root("arith.subi") -> ^1
 // CHECK-NEXT:     ^16:
-// CHECK-NEXT:       pdl_interp.switch_operation_name of [[arg0]] to ["foo.op", "bar.op"](^1, ^0) -> ^2
+// CHECK-NEXT:       pdl_interp.switch_operation_name of %arg5 to ["foo.op", "bar.op"](^1, ^17) -> ^2
```

Comment:
?
```suggestion
// CHECK-NEXT:       pdl_interp.switch_operation_name of %arg0 to ["foo.op", "bar.op"](^1, ^17) -> ^2
```

Reply:
Whoops, should be fixed. We need `[[arg0]]` because print-op-generic and custom assembly behave differently if I recall correctly.